cristin-person/src/test/java/no/unit/nva/cristin/person/update/UpdateCristinPersonHandlerTest.java

```diff
@@ -187,8 +196,44 @@ void shouldReturnBadRequestWhenInvalidIdentifier() throws IOException {
         assertThat(gatewayResponse.getBody(), containsString(ErrorMessages.invalidPathParameterMessage(PERSON_ID)));
     }
 
-    private static String randomIntegerAsString() {
-        return String.valueOf(randomInteger());
+    @Test
+    void shouldReturnNoContentResponseWhenCallingHandlerWithAllowedJsonValuesAndUserIsAuthenticatedAsHimself()
```

Comment:
AsHimself — is this necessary? Is the user not just authenticated? Either way, use gender-non-specific themselves. Perhaps this is more to do with matching user in token and elsewhere? Not sure what the intent is

Reply:
A user can either act as themselves, or as an user administrator. Therefore i specify themselves to differentiate between changing own data and changing others data